openpype/hosts/aftereffects/plugins/load/load_file.py

```diff
@@ -45,16 +58,17 @@ def load(self, context, name=None, namespace=None, data=None):
         path = path.replace("\\", "/")
 
         frame = None
+
         if '.psd' in path:
             import_options['ImportAsType'] = 'ImportAsType.COMP'
-            comp = stub.import_file_with_dialog(path, stub.LOADED_ICON + comp_name)
+            comp = stub.import_file_with_dialog(path, stub.LOADED_ICON + comp_name, fps)
```

Comment:
fps should be added in an `import_options` object argument, not as a standalone option argument of the function